### ISSUE TITLE
Unable to open the field manager and an error while using the register partial

### ISSUE THREAD

**Issue by noxify** [ShawnClake/UserExtended#114]
Hi,

i have installed the package and it seems that it's working...

But now I tried to add a new field, because of an error in the frontend...

![bildschirmfoto 2017-04-20 um 00 59 46](https://cloud.githubusercontent.com/assets/521777/25205776/b08b57cc-2564-11e7-8212-9db56cdf105f.png)

The CMS page looks like this:
![bildschirmfoto 2017-04-20 um 01 00 53](https://cloud.githubusercontent.com/assets/521777/25205817/e78e5238-2564-11e7-97ba-65fa56884332.png)

In the backend i got the following error while trying to access the field manager...

![bildschirmfoto 2017-04-20 um 01 02 32](https://cloud.githubusercontent.com/assets/521777/25205844/11c4ae58-2565-11e7-9423-095aab267613.png)

My installed packages are:
![bildschirmfoto 2017-04-20 um 01 03 08](https://cloud.githubusercontent.com/assets/521777/25205892/480bc938-2565-11e7-956a-e25e83f45c25.png)

My environment is:

* OSX with MAMP
* PHP 7.1
* mySQL 5.6

I have a backend user and a frontend user with the same email & username.

I have no clue what's wrong, because it seems that i'm the only one with this error :D

Anyway... i hope you can help me to solve this problem.

Thanks!

Greets,
Marcus

**Comment by ShawnClake** [comment]
Hey Marcus,

I pushed a potential fix for this a few hours ago. Let me know if this solved both problems.

Thanks,
Shawn

**Comment by noxify** [comment]
It seems that everything is working fine, both pages are now available.

But while adding a new field, i got a error in the web console:

![bildschirmfoto 2017-04-20 um 19 40 31](https://cloud.githubusercontent.com/assets/521777/25244511/5b9b03aa-2601-11e7-950c-95d912d0103d.png)


STRs:
* Opened BE as admin
* move to field manager
* one field (nickname) is shown in the list
* click on "add"
* Fill the following fields:
** Enabled: true
** Name: test
** Code: test
** Validating: <default>
** Placeholder: test
* Click on create

Expected: The new field "test" is shown in the list
Current: Only the nickname field is shown, an error is shown in the web console

**Comment by ShawnClake** [comment]
Hey Marcus,

Good to hear the first problem is fixed. The second problem is two fold. The Javascript errors have been fixed in the latest update I just pushed. 

However, the field seemingly not saving could be the result of validation failing. Sorry about that, the validation failures don't display an error message in this release, but in 2.2.00 which should release within the next week, that's been fixed. The validation currently requires the name, code, and description fields to have content inside of them. From your STR's it appears you left the description field blank which would cause the validation to fail and thus the model wouldn't save.

I hope this helps, thanks
Shawn
